adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -712,7 +689,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord]) extends Serializabl
    *
    * @return SingleReadBuckets with primary, secondary and unmapped reads
    */
-  def adamSingleReadBuckets(): RDD[SingleReadBucket] = {
+  def groupReads(): RDD[SingleReadBucket] = {
```

Comment:
> Since it is grouping reads together by the original sequencing fragment, perhaps ```groupReadsByFragment```?

+1

> In any case, perhaps do you want to ```s/SingleReadBucket/SingleRead/g``` or something like that?

I think I'll wait, I may want to propose other schema/class name changes in a separate pull request.